.semaphore/semaphore.yml

```diff
@@ -67,14 +67,14 @@ blocks:
             - echo "eval of CMD is completed";
             - ./.ci/validation.sh git-diff
 
-        - name: No Error Test (openjdk17, fast pool)
```

Comment:
nice its related to 21 migration mentioned in:
- https://github.com/checkstyle/checkstyle/pull/16980